esp/services/ws_ecl/ws_ecl_service.cpp

```diff
@@ -2051,9 +2062,9 @@ void CWsEclBinding::sendRoxieRequest(const char *target, StringBuffer &req, Stri
         ep = conn->nextEndpoint();
 
         Owned<IHttpClientContext> httpctx = getHttpClientContext();
-        StringBuffer url("http://");
+        WsEclSocketFactory *roxieConn = static_cast<WsEclSocketFactory*>(conn);
+        StringBuffer url(roxieConn->isTlsService() ? "https://" : "http://");
         ep.getIpText(url).append(':').append(ep.port ? ep.port : 9876).append('/');
```

Comment:
Not new but same comment re port constant

Reply:
I changed the others but this one could affect baremetal, so I'd rather leave it, unless you want me to investigate further.